app/models/event.rb

```diff
@@ -18,4 +19,17 @@ def data
       status: status
     }
   end
+
+  private
+
+  def date_validation
+    if !self[:finish_date].nil? && !self[:start_date].nil?
```

Comment:
`if !self[:finish_date].nil? && !self[:start_date].nil?`  -> `return if finish_date.blank? && start_date.blank?`
You do not need use `self`  .

Reply:
> `if !self[:finish_date].nil? && !self[:start_date].nil?` -> `return if finish_date.blank? && start_date.blank?`
> You do not need use `self` .

Validation doesn't work without `self`